powerauth-webflow-client/pom.xml

```diff
@@ -113,10 +123,6 @@
                             <groupId>org.projectlombok</groupId>
                             <artifactId>lombok</artifactId>
                         </exclude>
-                        <exclude>
```

Comment:
Are you sure about this? Is `servlet-api` really not included in `/WEB-INF/lib-provided`?